packages/expo-cli/src/commands/__tests__/publish-info-test.ts

```diff
@@ -0,0 +1,119 @@
+import { vol } from 'memfs';
+import { ApiV2 } from '@expo/xdl';
+
+import { getPublicationDetailAsync, getPublishHistoryAsync } from '../utils/PublishUtils';
+
+jest.mock('fs');
+jest.mock('resolve-from');
+jest.mock('ora', () =>
+  jest.fn(() => {
+    return {
+      start: jest.fn(() => {
+        return { stop: jest.fn(), succeed: jest.fn() };
+      }),
+    };
+  })
+);
+jest.mock('@expo/xdl', () => {
+  const user = {
+    kind: 'user',
+    username: 'test-username',
+    nickname: 'test-nickname',
+    userId: 'test-id',
+    picture: 'test-pic',
+    currentConnection: 'Username-Password-Authentication',
+    sessionSecret: 'test-session-secret',
+  };
+  const pkg = jest.requireActual('@expo/xdl');
+  pkg.UserManager.getCurrentUserAsync = jest.fn(() => user);
```

Comment:
this will mutate the actual @expo/xdl module can could lead to some confusing errors - when tests are run in parallel there will be failures in random places depending on the load order of the tests relative to this test.

a safer way to do this is:

```js
return {
  ...pkg,
  UserManager: {
    ...pkg.UserManager,
   getCurrentUserAsync: jest.fn(() => user),
  },
  // etc..
}
```